tools/protovalidate-conformance/internal/results/set.go

```diff
@@ -53,17 +54,20 @@ func (set *Set) Print(w io.Writer) {
 		(*SuiteResults)(suite).Print(writer)
 	}
 	res := resultLabel(set.Failures == 0)
-	_, _ = fmt.Fprintf(writer, "%s (failed: %d, passed: %d, total: %d)\n",
-		res, set.Failures, set.Successes, set.Failures+set.Successes)
+	_, _ = fmt.Fprintf(writer, "%s (failed: %d, skipped: %d, passed: %d, total: %d)\n",
+		res, set.Failures, set.ExpectedFailures, set.Successes, set.Failures+set.Successes+set.ExpectedFailures)
 	_ = writer.Flush()
 }
 
 type SuiteResults harness.SuiteResults
 
 func (suite *SuiteResults) AddCase(res *harness.CaseResult, verbose bool) {
-	if res.Success {
+	switch {
+	case res.Success && !res.ExpectedFail:
```

Comment:
One more consistency change.

`Success` is to `Failure`, as `Successes` is to `Failures`

In that spirit, can we rename the fields to either of these two sets (in pseudo code below)(whichever is preferred):

```
CaseResult {
  Success
  Failure
  ExpectedFailure
}
```
or
```
CaseResult {
  Successes
  Failures
  ExpectedFailures
}
```